.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+node_modules
```

Comment:
I notice you have `node_modules` in here twice - can you talk through what having it twice does different from once?